File: Lista_2/Exercicios_SALA-lista-2/EX9-Divisao_4_5-s/EX9-s.c
#include <stdio.h>

int main() {
    int n1, n2, n3;
    scanf("%d", &n1);
    scanf("%d", &n2);
    scanf("%d", &n3);

    if (n1 % 4 == 0) {
        printf("%d é divisível por 4\n", n1);
    }
    if (n1 % 5 == 0) {
        printf("%d é divisível por 5\n", n1);
    }
    if (n2 % 4 == 0) {
        printf("%d é divisível por 4\n", n2);
    }
    if (n2 % 5 == 0) {
        printf("%d é divisível por 5\n", n2);
    }
    if (n3 % 4 == 0) {
        printf("%d é divisível por 4\n", n3);
    }
    if (n3 % 5 == 0) {
        printf("%d é divisível por 5\n", n3);
    }

    return 0;
}

File: Lista_2/Exercicios_SALA-lista-2/EX9-Divisao_4_5-s/EX9-s.py
n = []
n.append(int(input()))
n.append(int(input()))
n.append(int(input()))
L2 = []
L3 = []
for i in n:
    if i % 4 == 0:
        L2.append(i)
    if i % 5 == 0:
        L3.append(i)
print(f"números divisíveis por 4: {L2}")
print(f"números divisíveis por 5: {L3}")
